samples/Server/package.json

```diff
@@ -33,7 +33,7 @@
     "http-errors": "^1.6.3",
     "morgan": "^1.9.1",
     "node-fetch": "2.6.7",
-    "@azure/communication-rooms": "1.0.0-beta.1"
+    "@azure/communication-rooms": "1.1.0-beta.1 || ^1.1.0"
```

Comment:
Do we need to do the rush common versions here?

Reply:
We don't need to because rooms is not using for the ui library. It's only used by the Server.